src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -19,5 +21,10 @@ public class CoreFeatureConfiguration
         /// Gets or sets a value indicating whether Transaction is enabled or not.
         /// </summary>
         public bool SupportsTransaction { get; set; }
+
+        /// <summary>
+        /// Gets or sets the default value for IncludeTotal in search bundles
+        /// </summary>
+        public TotalType BundleIncludeTotal { get; set; } = TotalType.None;
```

Comment:
nit: since the other features start with a verb, maybe this should be `IncludeTotalInBundle`.